packages/console/src/monaco/MonacoUtils.ts

```diff
@@ -450,6 +451,26 @@ class MonacoUtils {
       KeyCodeUtils.fromString(keyValue)
     );
   }
+
+  static getProvideLinks = () => (model: monaco.editor.ITextModel) => {
```

Comment:
No need to curry the method, just have it as a regular method:
```suggestion
  static provideLinks(model: monaco.editor.ITextModel) {
```